hack/build/setup-pac-integration.sh

```diff
@@ -48,19 +52,21 @@ setup-pac-app() (
         payload="{ \"iss\": $PAC_GITHUB_APP_ID, \"iat\": ${now}, \"exp\": $((now+10)) }"
 
         webhook_secret=$(openssl rand -hex 20)
+        token=$(sign rs256 "$payload" "$(echo "$PAC_GITHUB_APP_PRIVATE_KEY" | base64 -d)")
+        # return  sprayproxy host in a loop until it is available
+        while ! kubectl get -n sprayproxy route/sprayproxy-route &>/dev/null; do
+                echo "Waiting for sprayproxy route to be available"
```

Comment:
must go to stderr, otherwise it's retuned as the webhook secret itself and secret is created this way:
```
Waiting for sprayproxy route to be available
Waiting for sprayproxy route to be available
Waiting for sprayproxy route to be available
Waiting for sprayproxy route to be available
Waiting for sprayproxy route to be available
Waiting for sprayproxy route to be available
Waiting for sprayproxy route to be available
Waiting for sprayproxy route to be available
Waiting for sprayproxy route to be available
Waiting for sprayproxy route to be available
Waiting for sprayproxy route to be available
Waiting for sprayproxy route to be available
Waiting for sprayproxy route to be available
$TOKEN
```
But anyway I am missing configuration of spray proxy to point to PAC. Maybe waiting here for the sprayproxy route is not needed and can be hardcoded.

Reply:
>must go to stderr

fixed

Reply:
> configuration of spray proxy to point to PAC

@Michkov regarding this, I am not sure where it's the best place. When e2e CI is launched , it needs to register Pac in a permanent Sprayprox. When e2e CI finish, we want to unregister the pac url from the Sprayprox. So maybe e2e repo is the best place to implement registering/unregistering Pac to sprayproxy.

Reply:
that echo still goes to stdout

Reply:
fixed